.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: c
+sudo: required
+dist: trusty
+
+install:
+        - sudo apt-add-repository -y ppa:team-gcc-arm-embedded/ppa
+        - sudo apt-get update
+        - sudo apt-get install -yy gcc-arm-embedded
+
+script:
+        - cd efm32hg-blinky
+        - make deps
```

Comment:
There's one step here that isn't automated and I think should be added to the deps section: the Gecko_SDK has the linker script's FLASH ORIGIN at 0x0, and it needs to be changed to 0x4000, and the LENGTH needs to be changed to 0xC000.
The EFM32(HG) bootloader resides between 0x0 and 0x3FFF, and when uploading applications, it stores them at 0x4000 onwards.
Without this fix, binaries will successfully compile, but won't successfully run.
Maybe this should be a second step as part of the deps target?